src/transaction/log_page_buffer.c

```diff
@@ -3272,6 +3272,8 @@ logpb_append_prior_lsa_list (THREAD_ENTRY * thread_p, LOG_PRIOR_NODE * list)
   assert (log_Gl.prior_info.prior_flush_list_header == NULL);
   log_Gl.prior_info.prior_flush_list_header = list;
 
+  // TODO: send is synched here; is this on purpose?
```

Comment:
Should this be a point of attention? Maybe not now, later on...

Reply:
I don't really get the todo; what does it mean by send is synched?